From fb40907156088af74bd02fdb82e8de76e97048a7 Mon Sep 17 00:00:00 2001
From: David Andrey <david.andrey@netmodule.com>
Date: Thu, 29 Nov 2012 13:16:29 +0100
Subject: [PATCH 09/12] ARM: xilinx: Micrel PHY: disabled PAUSE

No Gbit link can be auto-negotiated with PAUSE
sym and asym enabled, for KSZ9021 too

Signed-off-by:David Andrey david.andrey@netmodule.com
---
 drivers/net/phy/micrel.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/drivers/net/phy/micrel.c b/drivers/net/phy/micrel.c
index 99d0e33..dea040a 100644
--- a/drivers/net/phy/micrel.c
+++ b/drivers/net/phy/micrel.c
@@ -176,8 +176,7 @@ static struct phy_driver ksz9021_driver = {
 	.phy_id		= PHY_ID_KSZ9021,
 	.phy_id_mask	= 0x000ffffe,
 	.name		= "Micrel KSZ9021 Gigabit PHY",
-	.features	= (PHY_GBIT_FEATURES | SUPPORTED_Pause
-				| SUPPORTED_Asym_Pause),
+	.features	= PHY_GBIT_FEATURES, /* @NetModule, da, PAUSE disabled, else Gbit not working */
 	.flags		= PHY_HAS_MAGICANEG | PHY_HAS_INTERRUPT,
 	.config_init	= kszphy_config_init,
 	.config_aneg	= genphy_config_aneg,
-- 
1.7.4.1

